delta/streaming/writers.py

```diff
@@ -131,6 +131,23 @@ def put_data(self, data_class, attrs: dict):
 
         return None
 
+    # (2020/11) jyc: this is a temporary hack to write numpy data directly.
```

Comment:
See comment above. Any class with a 'data' member can be passed to writer.put_data

Reply:
Thanks. Removed the temporary fix.